css/index.css

```diff
@@ -1 +1,393 @@
+    padding: 1.2%;
+    width: 36%;
+}
+
+.btn-learn {
+    border-radius: 6px;
+    font-size: 12px;
+    margin-top: 26px;
+    padding: 8px;
+    width: 126px;
+}
+
+.feature-content {
+    display: flex;
+    margin-bottom: 40px;
+}
+
+.feature-wrapper {
+    padding-right: 2.3%;
+    width: 50%;
+}
+
+.middle-img {
+    width: 100%;
+    margin-bottom: 40px;
+}
+
+.service-content {
+    display: flex;
+    margin-bottom: 30px;
+    border-bottom: 2px solid;
+}
+
+.service-wrapper {
+    margin-bottom: 30px;
+    padding-right: 1.3%;
+    width: 33.4%;
+}
+
+.contact-container {
+    font-size: 1.6rem;
+}
+
+/* Below: service page
+row-column section flexbox */
+
+/*.services-header-img {*/
+/*margin-top: 50px;*/
+/*width: 100%;*/
+/*}*/
+
+/*.services-page-top-section {*/
+/*margin-top: 30px;*/
+/*margin-bottom: 40px;*/
+/*}*/
+
+/*.services-page-h1 {*/
+/*font-size: 45px;*/
+/*}*/
+
+/*.row {*/
+/*display: flex;*/
+/*flex-direction: row;*/
+/*flex-wrap: wrap;*/
+/*width: 100%;*/
+/*}*/
+
+/*.column {*/
+/*display: flex;*/
+/*flex-direction: column;*/
+/*flex-basis: 100%;*/
+/*flex: 1;*/
+/*border: 1px solid;*/
+/*background-color: #ededed;*/
+/*margin-bottom: 40px;*/
+/*padding: 20px;*/
+
+/*}*/
+
+/*.column-left {*/
+/*margin-left: 20px;*/
+/*}*/
+
+/*.column-right {*/
+/*margin-right: 20px;*/
+/*}*/
+
+/*.services-page-divider {*/
+/*border-style: solid;*/
+/*height: 3px;*/
+/*background-color: black;*/
+/*}*/
+
+/*.services-page-last-section {*/
+/*display: flex;*/
+/*justify-content: space-around;*/
+/*}*/
+
+/*!*.facts-section-wrapper {*!*/
+/*!*width: 400px;*!*/
+/*!*}*!*/
+
+/*.facts-list-wrapper {*/
+/*margin-right: 20px;*/
+/*}*/
+
+/*.facts-list {*/
+/*list-style-type: disc;*/
+/*margin-top: 30px;*/
+/*padding-left: 0;*/
+
+/*}*/
+
+/*.facts-list-items {*/
+/*margin-bottom: 20px;*/
+/*list-style-position: inside*/
+/*}*/
+
+/* Above: service page row-column */
+
+.contact {
+    margin-bottom: 20px;
+}
+
+/* mail and phone emoji below */
+
+a[href^="mailto"]::before {
+    content: "📧 ";
+    text-decoration: none;
+}
+
+a[href^="tel"]::before {
+    content: "📞 ";
+    text-decoration: none;
+}
+
+.link {
+    color: black;
+    text-decoration: none;
+}
+
+.footer {
+    margin-bottom: 40px;
+    margin-top: 40px;
+    text-align: center;
+}
+
+/*tablet view media query*/
+
+@media(max-width: 800px) {
+    .header-container-hidden {
+        display: none;
+    }
+
+    .main-container {
+        padding-left: 1.3%;
+        padding-right: 1.3%;
+    }
+
+    nav .main-container {
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .nav-item-wrapper {
+        display: flex;
+        justify-content: space-around;
+        width: 100%;
+    }
+
+    .logo-wrapper {
+        margin: 20px 0;
+        width: auto;
+    }
+}
+
+/*mobile view media query*/
+
+@media(max-width: 500px) {
+
+    nav {
+        height: 372px;
```

Comment:
remember what I said about setting heights. Look at what happens when you change the browser settings to extra large font